.github/workflows/ci.yml

```diff
@@ -333,7 +333,6 @@ jobs:
         id: tests
         run: |
           $MAVEN test ${MAVEN_TEST} -pl '
-            !:trino-accumulo,
```

Comment:
`set-session.md` still refers to Accumulo connector.

Reply:
I know .. hence its still a draft .. i will add another commit that changes that whole section.

Reply:
Updated that example now to use Iceberg.